### PR TITLE
Regal rat antag

### PR DESCRIPTION
**Pull request by ghost**
# Document the changes in your pull request

Ports regal rat antagonist from tgstation. It is a very fun antag, that needs to convert rats in mantaince and do funny rat shit. He can eat cheese to heal, and lick human food to convert crewmembers.

[1](https://github.com/tgstation/tgstation/pull/65890/files)
[2](https://github.com/tgstation/tgstation/pull/49884)
# Wiki Documentation

Port this wiki page: [link](https://tgstation13.org/wiki/Rat_King)

# Changelog

:cl: Author: ArcaneMusic
rscadd: Adds new Regal rat antagonist.
rscadd: Adds royal cheese and it's crafting recipe.
imageadd: Adds sprites for Rat King and royal cheese.
E
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
OH REALLY FUCKING MERGE CONFLICTS AGAIN

**Comment by ghost** [comment]
Bruh ok

**Comment by Cartlord** [comment]
Are you aware that "rat kings" are a real thing that exist? Is this anything like 'em?

**Comment by ghost** [comment]
> Are you aware that "rat kings" are a real thing that exist? Is this anything like 'em?

Yogstation doesnt have rat kings/regal rats, if i correctly understand your question.

**Comment by ghost** [comment]
This is temporary unfinished due to unexpected bugs

**Comment by ghost** [comment]
Okay, finally fixed. Can someone approve checks?

**Comment by ghost** [comment]
Please checks

**Comment by ArcaneMusic** [comment]
> Are you aware that "rat kings" are a real thing that exist? Is this anything like 'em?

When I made regal rats in 2020, this was brought up as well. That's why they're regal rats and not called rat kings. 👍

**Comment by ghost** [comment]
Understand…

**Comment by ghost** [comment]
> Are you aware that "rat kings" are a real thing that exist? Is this anything like 'em?

Okay now i actually understand your question. Yes, i know that such thing like “rat king” exists at real live, and this antagonist isnt like it. It is actually just a mutated rat with some fun powers, and it doesnt have anything same with the rat king from real live.

**Comment by nmajask** [comment]
Credit the original author of the PR and link it if you can

**Comment by ghost** [comment]
> Credit the original author of the PR and link it if you can

Credited. Also PR's are alredy linked.

**Comment by missatessatessy** [comment]
Suggestion: put the code for the royal cheesewheel object into the snacks_cheese.dm file not the snacks_other file

**Comment by missatessatessy** [comment]
Oh additionally you should make the recipe for the royal cheese to use cheddar cheese wheel specifically, or create an individual sprite for every cheese we have, otherwise it will be weird if a block of feta suddenly turns into a cheesewheel ;)

**Comment by ghost** [comment]
Ok, but i cant do all of this now

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_misc.dm [hunk 1]
```diff
@@ -350,3 +350,15 @@
 	)
 	result = /obj/item/reagent_containers/food/snacks/candy_strawberry
 	subcategory = CAT_MISCFOOD
+
+
+/datum/crafting_recipe/food/royalcheese
+	name = "Royal Cheese"
+	reqs = list(
+		/obj/item/reagent_containers/food/snacks/store/cheesewheel/feta = 1,
```
Comment (tattax) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_misc.dm]:
```suggestion
		/obj/item/reagent_containers/food/snacks/store/cheesewheel/cheddar = 1,
```

**Comment by JamieD1** [comment]
Tessa needs to review.

**Comment by ghost** [comment]
Yes please

**Comment by ghost** [comment]
> Oh additionally you should make the recipe for the royal cheese to use cheddar cheese wheel specifically, or create an individual sprite for every cheese we have, otherwise it will be weird if a block of feta suddenly turns into a cheesewheel ;)

So i replaced feta in craft with cheddar, and moved the royal cheese into snacks_chease, so when you can review this?

**Review by nesisknesis — APPROVED** [review]

**Comment by ghost** [comment]
Checks also please

**Comment by ghost** [comment]
I am still waiting

**Review by JamieD1 — APPROVED** [review]

**Comment by ghost** [comment]
🥳

**Pull request merged**